swiftshield-Sources/XcodeProjectBuilder.swift

```diff
@@ -28,7 +28,7 @@ final class XcodeProjectBuilder {
         let arguments: [String] = [projectParameter, projectToBuild, "-scheme", schemeToBuild]
         let cleanTask = Process()
         cleanTask.launchPath = path
-        cleanTask.arguments = ["clean", "build"] + arguments
+        cleanTask.arguments = ["clean", "build"] + arguments + ["-sdk", "iphonesimulator"]
```

Comment:
Is this necessary on Xcode 11? What happens if you have a watchOS target on your app?

Reply:
Without `-sdk` parameter, the build will be failed with error
```
Code Signing Error: "SwiftShieldExample" requires a provisioning profile. Select a provisioning profile in the Signing & Capabilities editor.
Code Signing Error: Code signing is required for product type 'Application' in SDK 'iOS 13.0'
```
Of course most of projects should contain these items. But if we want to obfuscate the demo project and without code sign and profiles, this job will not be successful.

Reply:
OK, I have reverted these changes.